packages/shared/src/components/tooltips/ChangelogTooltip.spec.tsx

```diff
@@ -364,4 +365,31 @@ describe('ChangelogTooltip component', () => {
 
     expect(updateAlerts).toHaveBeenCalled();
   });
+
+  it('calls toggleUpvote on upvote button click', async () => {
+    const client = new QueryClient();
+    const toggleUpvote = jest.fn();
+
+    jest.spyOn(hooks, 'useChangelog').mockImplementation(() => ({
+      toggleUpvote,
+      isAvailable: true,
+      latestPost: defaultPost,
+      dismiss: jest.fn(),
+    }));
```

Comment:
Do you need this part, I'm not 100% sure, but since you mock the function above it feels you could omit it.

Reply:
The function above is just a mock not connected to the useChangelog toggleUpvote, line 374 mocks the real toggleUpvote to be our mock toggleUpvote. As for the rest of the props TS complains if those are not added.